src/serve_rendered.js

```diff
@@ -294,6 +294,8 @@ const extractMarkersFromQuery = (query, options, transformer) => {
       // When we encounter a remote icon check if the configuration explicitly allows them.
     } else if (options.allowRemoteMarkerIcons !== true) {
       continue;
+    } else if (options.allowInlineMarkerImages !== true) {
```

Comment:
We probably only want this option to apply when using a data URL. Would maybe suggest a refactor to something like:
```js
const isRemoteURL = iconURI.startsWith('http://') || iconURI.startsWith('https://');
const isDataURL = iconURI.startsWith('data:');

if (!(isRemoteURL || isDataURL)) {
  ...
} else if (isRemoteURL && options.allowRemoteMarkerIcons !== true) {
  continue;
} else if (isDataURL && options.allowInlineImarkerImages !== true) {
  continue;
}
```